test/integration/raft_test.lua

```diff
@@ -312,8 +312,8 @@ g.test_kill_master = function()
     t.assert_equals(res.status, 200)
     t.assert_equals(res.json, {})
 
-    start_server('storage-3')
     kill_server('storage-2')
+    start_server('storage-3')
```

Comment:
Why is this change necessary?

Reply:
I think we discussed it before on call. Changing the positions of kill/start will reduce noise from raft and probably make the test more stable. But it's not a necessary change and I can remove it. What do you think?

Reply:
But storage-3 is 'voter', it can't interfere with elections. I think this shouldn't affect the test stability, unless I forgot something else we discussed.

Let's remove this change.

Reply:
Done.